src/net.h

```diff
@@ -1170,18 +1170,22 @@ class CConnman
     // Addrman functions
     /**
      * Return all or many randomly selected addresses, optionally by network.
+     * This function does not use an address response cache and should only be
```

Comment:
Even though the PR is quite simple, I'd separate the comment-changing-commits from the refactoring commits. I guess it may not make a lot of sense doing the rename with a scripted diff since it might confuse the two similarly named methods.